react/javascript/src/styles/styles.scss

```diff
@@ -28,12 +28,6 @@ $summary-border-color: rgba(0, 0, 0, 0.1);
   }
 }
 
-// TODO encapsulate in a component
-.cucumber-children {
```

Comment:
I think this may be used in more places than the `<Feature>` component?

Reply:
Fixed in 53cc7f6b27720d4d9aa9fff14cde50bf7097f306 - added a `<Children>` component